packages/cli/src/commands/dev-contracts.ts

```diff
@@ -142,8 +143,10 @@ const commandModule: CommandModule<Options, Options> = {
       // Build the contracts
       await forge(["build"]);
 
-      // Run typechain to rebuild typescript types for the contracts
-      await worldtypes();
+      // Generate TS-friendly ABI files
+      // We rebuild into a separate dir to have a clean set of ABIs without test/script contracts
```

Comment:
is this comment still up to date?

Reply:
yep, still relevant

Reply:
more context:

We need forge to build everything to run things like tests, scripts, etc. But for a clean ABI dir, we want to ignore these. It's tricky to use the same target dir (e.g. `out`) because then tests won't find the build files they need to run. So we use a separate dir just for ABIs.

Since abi-ts can now target/build a specific ABI file or glob, we could simplify this to just a single output dir and only build IWorld ABI, but has trade-offs in that it's a bit harder for users to expand that if they want to include more ABIs in their TS code.